lib/tasks/openapi_generate.rake

```diff
@@ -1,32 +1,5 @@
 require 'insights/api/common/open_api/generator'
 class OpenapiGenerator < Insights::API::Common::OpenApi::Generator
-  def generator_blacklist_allowed_attributes
-    @generator_blacklist_allowed_attributes ||= {
-      :tenant_id => ['Authentication', 'Application', 'Endpoint', 'Source'].to_set.freeze
-    }
-  end
```

Comment:
I started looking into having this stick around for v1 but not v2 and found that `Insights::API::Common::OpenApi::Generator` only updates the latest API anyway.  This means that to update an older API you have to comment out all newer APIs in `config/routes.rb`.  Not great but I'm not sure that I want to solve that right now.

Reply:
Yeah, maybe we should allow for an ENV variable to select the version you want to run the generator for.